packages/lodestar-cli/src/cmds/beacon/cmds/init/init.ts

```diff
@@ -1,47 +1,49 @@
 import path from "path";
-import {Arguments} from "yargs";
 import deepmerge from "deepmerge";
-
-import {rootDir} from "../../../../options";
-import {beaconDir} from "../../options/beaconDir";
-import {IBeaconArgs} from "../../options";
+import {globalOptions} from "../../../../options";
+import {IBeaconOptions} from "../../options";
 import {mkdir} from "../../../../util";
 import {initPeerId, initEnr, readPeerId} from "../../../../network";
-import {initBeaconConfig} from "../../config";
+import {initBeaconConfig, mergeConfigOptions} from "../../config";
 import {getTestnetConfig, downloadGenesisFile, fetchBootnodes} from "../../testnets";
+import {getBeaconPaths} from "../../paths";
 
 /**
  * Initialize lodestar-cli with an on-disk configuration
  */
-export async function init(args: Arguments<IBeaconArgs>): Promise<void> {
+export async function initHandler(options: IBeaconOptions): Promise<void> {
+  options = mergeConfigOptions(options);
```

Comment:
I don't think we want to merge from `configFile` here, since thats where we're writing out the configuration being created below in `initBeaconConfig`.

Reply:
Right, actually `options = mergeConfigOptions(options)` could be not called at all in the init handler? All the options actually necessary are computed with defaults in `beaconPaths`. No option in `options` used needs to be assigned a default.